src/settings.js

```diff
@@ -190,8 +190,17 @@ var OUTLINING_LIMIT = 0; // A function size above which we try to automatically
 
 var AGGRESSIVE_VARIABLE_ELIMINATION = 0; // Run aggressiveVariableElimination in js-optimizer.js
 var SIMPLIFY_IFS = 1; // Whether to simplify ifs in js-optimizer.js
-var POINTER_MASKING = 0; // Whether to mask pointers (experimental optimization trying to reduce VM bounds checks).
-                         // When using this option, TOTAL_MEMORY must be a power of 2.
+
+// Whether to mask pointers to help avoid bounds checks.  When using this
+// option, TOTAL_MEMORY must be a power of 2.
```

Comment:
please keep the comments in the same format as on the other options, i.e. `var X = default; // comments go here`

please also keep the comment that this is an experimental optimization.